openpype/hosts/photoshop/api/extension/host/index.jsx

```diff
@@ -263,6 +263,42 @@ function getActiveDocumentFullName(){
     };
 }
 
+function getActiveDocumentFormatResolution(){
+    /**
+     *   Returns a dict with the resolution dimenssion of the current doc
+     * */
+    if (documents.length == 0){
+        return null;
```

Comment:
Based on my comment with the `json.loads(res)` on the `ws_stub` file, this should probably be `return {}`

Reply:
corrigé